src/Microsoft.Data.SqlClient/src/Microsoft/Data/Sql/SqlDataSourceEnumerator.cs

```diff
@@ -0,0 +1,198 @@
+//------------------------------------------------------------------------------
+// <copyright file="SqlDataSourceEnumerator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <owner current="true" primary="true">Microsoft</owner>
+// <owner current="true" primary="false">Microsoft</owner>
+//------------------------------------------------------------------------------
```

Comment:
If this is going to be published with the rest of the library code, the header needs to be changed to open source header.